### ISSUE TITLE
kinetic to noetic

### ISSUE THREAD

**Issue by bhavyawadhwa856** [osrf/rvizweb#53]
Thanks for providing the great RVizWeb. However, it is only running fine with kinetic version. Could you please provide me suitable and running Rvizweb for noetic. It would be a grate help.

**Comment by hadieh-ayman** [comment]
yes! I also would like to use this repo, I hope it becomes available for noetic soon.
